File: week-2/exercicios/media.cpp
#include<iostream>

int main() {
  int nota_1, nota_2, nota_3, nota_4;
  
  std::cout << "Digite a primeira nota ";
  std::cin >> nota_1;

  std::cout << "Digite a segunda nota";
  std::cin >> nota_2;

  std::cout << "Digite a terceira nota";
  std::cin >> nota_3;

  std::cout <<"Digite a quarta nota";
  std::cin >> nota_4;

  float media = (nota_1 + nota_2 + nota_3 + nota_4) / 4;
  std::cout << "A media aritmetica e " << media;
  
  return 0;
}

File: week-2/exercicios/media.py
"""
Exercicio 2
Faça um programa em Python que receba quatro notas, calcule e imprima a média aritimetica.
"""

primeira_nota = int(input("Digitte a primeira nota: "))
segunda_nota = int(input("Digite a segunda nota: "))
terceira_nota = int(input("Digite a terceira nota: "))
quarta_nota = int(input("Digite a quarta nota: "))

media = float((primeira_nota + segunda_nota + terceira_nota + quarta_nota) / 4)
print('A média aritmética é',media)
